packages/headless/src/api/search/search-api-client.test.ts

```diff
@@ -41,6 +46,9 @@ describe('search api client', () => {
       logger,
       renewAccessToken,
       preprocessRequest: NoopPreprocessRequestMiddleware,
+      preprocessSearchResponseMiddleware: NoopPreprocessSearchResponseMiddleware,
```

Comment:
Instead of `preprocess`, should these be `postprocess` ? 

I understand that, by using those, you are `preprocessing the response before it gets updated in the state`, but ... post process seems more natural to me ?